Wikipedia/Code/WMFSearchViewController.m

```diff
@@ -278,9 +278,18 @@ - (void)setSeparatorViewHidden:(BOOL)hidden animated:(BOOL)animated {
 
 #pragma mark - Dismissal
 
-- (IBAction)didTapCloseButton:(id)sender {
+- (BOOL)didRequestDismiss {
```

Comment:
"did" usually indicates a delegate method. This is just an internal command - so simply `- (void)dismiss` would be an appropriate name.